build: Cleanup depends build system

hebasto opened the pull request:
This PR:
- removes non-existent `share/pkgconfig` path from `PKG_CONFIG_PATH`. This change, actually, make `PKG_CONFIG_PATH` unused in the depends build system
- removes `doc`,  `man` and empty directories from the built packages

hebasto committed:
build: Drop non-existent share/pkgconfig directory

hebasto committed:
build: Remove unneeded share/doc directory from expat package

hebasto committed:
build: Remove empty var/cache/fontconfig directory from fontconfig

hebasto committed:
build: Remove unneeded share/man directory from freetype package

hebasto committed:
build: Remove unneeded share/man directory from libXau package

hebasto commented:
#### Guix builds:
```
$ find guix-build-$(git rev-parse --short=12 HEAD)/output/ -type f -print0 | env LC_ALL=C sort -z | xargs -r0 sha256sum
cec6173401b3f810447fbde0ffe663d0ab06b78de6f9ca2849d6b84935f4b7ba  guix-build-9b64d1749f5b/output/aarch64-linux-gnu/SHA256SUMS.part
3dc8545307cd941a8c9ddea7185309e3965779cf675db8e20f053dc31071c2b7  guix-build-9b64d1749f5b/output/aarch64-linux-gnu/bitcoin-9b64d1749f5b-aarch64-linux-gnu-debug.tar.gz
2d0be71d3d9889389283b4049bd79725cb338ccc6b7ec368bbccb8fdeed0b3ef  guix-build-9b64d1749f5b/output/aarch64-linux-gnu/bitcoin-9b64d1749f5b-aarch64-linux-gnu.tar.gz
eb7cc6df4229c992f47534fa0ef9adde8b04f58a7aa8cb286964f841daf43903  guix-build-9b64d1749f5b/output/arm-linux-gnueabihf/SHA256SUMS.part
ae4db85be8d3320b42d0abecac16ff0cc6d7c24249db3ffb04915c97aac2f3df  guix-build-9b64d1749f5b/output/arm-linux-gnueabihf/bitcoin-9b64d1749f5b-arm-linux-gnueabihf-debug.tar.gz
e414b89fdd128f1f69422feb2684259e51d91251435491144c011978acf61b17  guix-build-9b64d1749f5b/output/arm-linux-gnueabihf/bitcoin-9b64d1749f5b-arm-linux-gnueabihf.tar.gz
b7294a8f00bb84347db24f712701fea3ae2edd5874431cb6e3994a1cf8035b0a  guix-build-9b64d1749f5b/output/dist-archive/bitcoin-9b64d1749f5b.tar.gz
37c57e1b968cfc9e429328f6c6f204d2a51b5f8e0495685f96d84e3a56636e8b  guix-build-9b64d1749f5b/output/powerpc64-linux-gnu/SHA256SUMS.part
a3b3cf649f0bb05ec5701ec4a52cbd577f23a102879798a0b1fa855aee366018  guix-build-9b64d1749f5b/output/powerpc64-linux-gnu/bitcoin-9b64d1749f5b-powerpc64-linux-gnu-debug.tar.gz
98e7102288b4e4d52f59bf26e659ab6c9730b3fa12e97a04b037d6a1ea921198  guix-build-9b64d1749f5b/output/powerpc64-linux-gnu/bitcoin-9b64d1749f5b-powerpc64-linux-gnu.tar.gz
269d0ffff3eaa594be82dfc602090c13787faba2c26d118114788d74f1c431a8  guix-build-9b64d1749f5b/output/powerpc64le-linux-gnu/SHA256SUMS.part
8b3e73086e320e043c9ea1459595e567f48d2d9e4432c30eb5c2d00d537a18c1  guix-build-9b64d1749f5b/output/powerpc64le-linux-gnu/bitcoin-9b64d1749f5b-powerpc64le-linux-gnu-debug.tar.gz
2d94840ccfe6bb09ab0aaf51d9d54ed0cbd66eb8957554ae7dcfecb70a27fd1b  guix-build-9b64d1749f5b/output/powerpc64le-linux-gnu/bitcoin-9b64d1749f5b-powerpc64le-linux-gnu.tar.gz
cfb42ec938809247195d57be6a9f5219464f9a4dbaf3ce2cd74d732e3b87935f  guix-build-9b64d1749f5b/output/riscv64-linux-gnu/SHA256SUMS.part
a61fb13768962f1dd3b76186f604a9b29cd024a7f9fed50471f6e2bd5431bf0c  guix-build-9b64d1749f5b/output/riscv64-linux-gnu/bitcoin-9b64d1749f5b-riscv64-linux-gnu-debug.tar.gz
bac86b9c157ce510bca4b737356b80133f0948feac4ed7b98ca03d4faa83b90e  guix-build-9b64d1749f5b/output/riscv64-linux-gnu/bitcoin-9b64d1749f5b-riscv64-linux-gnu.tar.gz
120cefb379ba0bbd88711dcce978a3168ac5a0044267ca67cc7d8dcd4a97a59c  guix-build-9b64d1749f5b/output/x86_64-apple-darwin18/SHA256SUMS.part
e2ac86fca93b2b066752a6cf1f4af36c037e7f3ac3b094e71ce12b0892c97aec  guix-build-9b64d1749f5b/output/x86_64-apple-darwin18/bitcoin-9b64d1749f5b-osx-unsigned.dmg
fdc27376db1fc1a10fce9c4ab19da47bd2693fefa7cf0716ca89e22793d07d32  guix-build-9b64d1749f5b/output/x86_64-apple-darwin18/bitcoin-9b64d1749f5b-osx-unsigned.tar.gz
2dea8736bbd9881bd5f2bb4303390d6be06fef1d6cb4669760361190e8a91976  guix-build-9b64d1749f5b/output/x86_64-apple-darwin18/bitcoin-9b64d1749f5b-osx64.tar.gz
a1bd5a21ac1314d74d99da6fac6609f5dc282bdac54cce94dd2722cccc401c9e  guix-build-9b64d1749f5b/output/x86_64-linux-gnu/SHA256SUMS.part
998fe0dd4b13ab7b930c58788e81be5c42841d61d37bdf9ad26607e84b285fdd  guix-build-9b64d1749f5b/output/x86_64-linux-gnu/bitcoin-9b64d1749f5b-x86_64-linux-gnu-debug.tar.gz
c4b8ad245a3ea230f1514d416b4c9f9ad12a7d17a800f63697e1c93271ae3df5  guix-build-9b64d1749f5b/output/x86_64-linux-gnu/bitcoin-9b64d1749f5b-x86_64-linux-gnu.tar.gz
dc7a5ca9fd16017330f2f668618342e3088e464b10381bd10763f85307d53b3f  guix-build-9b64d1749f5b/output/x86_64-w64-mingw32/SHA256SUMS.part
b4245aba901c9a223867d17cc440e0e6a15c6b5ec2934d2c149076235ae0efb7  guix-build-9b64d1749f5b/output/x86_64-w64-mingw32/bitcoin-9b64d1749f5b-win-unsigned.tar.gz
8005a4dc58c92c6e2b6018f62cc960e2b5fdd600f88fbfea10bb7fd4123c12ae  guix-build-9b64d1749f5b/output/x86_64-w64-mingw32/bitcoin-9b64d1749f5b-win64-debug.zip
1f5fcaea89614cd6107386e43a4679fd05ab0b0ce22ddb47d34313e65d275307  guix-build-9b64d1749f5b/output/x86_64-w64-mingw32/bitcoin-9b64d1749f5b-win64-setup-unsigned.exe
dcd8814debfc8dc7a90c86572fd753afea0c772ac34976692118af626b41d117  guix-build-9b64d1749f5b/output/x86_64-w64-mingw32/bitcoin-9b64d1749f5b-win64.zip
```

jarolrod commented:
GUIX hashes, mine match @hebasto:

```
find guix-build-$(git rev-parse --short=12 HEAD)/output/ -type f -print0 | env LC_ALL=C sort -z | xargs -r0 sha256sum

cec6173401b3f810447fbde0ffe663d0ab06b78de6f9ca2849d6b84935f4b7ba  guix-build-9b64d1749f5b/output/aarch64-linux-gnu/SHA256SUMS.part
3dc8545307cd941a8c9ddea7185309e3965779cf675db8e20f053dc31071c2b7  guix-build-9b64d1749f5b/output/aarch64-linux-gnu/bitcoin-9b64d1749f5b-aarch64-linux-gnu-debug.tar.gz
2d0be71d3d9889389283b4049bd79725cb338ccc6b7ec368bbccb8fdeed0b3ef  guix-build-9b64d1749f5b/output/aarch64-linux-gnu/bitcoin-9b64d1749f5b-aarch64-linux-gnu.tar.gz
eb7cc6df4229c992f47534fa0ef9adde8b04f58a7aa8cb286964f841daf43903  guix-build-9b64d1749f5b/output/arm-linux-gnueabihf/SHA256SUMS.part
ae4db85be8d3320b42d0abecac16ff0cc6d7c24249db3ffb04915c97aac2f3df  guix-build-9b64d1749f5b/output/arm-linux-gnueabihf/bitcoin-9b64d1749f5b-arm-linux-gnueabihf-debug.tar.gz
e414b89fdd128f1f69422feb2684259e51d91251435491144c011978acf61b17  guix-build-9b64d1749f5b/output/arm-linux-gnueabihf/bitcoin-9b64d1749f5b-arm-linux-gnueabihf.tar.gz
b7294a8f00bb84347db24f712701fea3ae2edd5874431cb6e3994a1cf8035b0a  guix-build-9b64d1749f5b/output/dist-archive/bitcoin-9b64d1749f5b.tar.gz
37c57e1b968cfc9e429328f6c6f204d2a51b5f8e0495685f96d84e3a56636e8b  guix-build-9b64d1749f5b/output/powerpc64-linux-gnu/SHA256SUMS.part
a3b3cf649f0bb05ec5701ec4a52cbd577f23a102879798a0b1fa855aee366018  guix-build-9b64d1749f5b/output/powerpc64-linux-gnu/bitcoin-9b64d1749f5b-powerpc64-linux-gnu-debug.tar.gz
98e7102288b4e4d52f59bf26e659ab6c9730b3fa12e97a04b037d6a1ea921198  guix-build-9b64d1749f5b/output/powerpc64-linux-gnu/bitcoin-9b64d1749f5b-powerpc64-linux-gnu.tar.gz
269d0ffff3eaa594be82dfc602090c13787faba2c26d118114788d74f1c431a8  guix-build-9b64d1749f5b/output/powerpc64le-linux-gnu/SHA256SUMS.part
8b3e73086e320e043c9ea1459595e567f48d2d9e4432c30eb5c2d00d537a18c1  guix-build-9b64d1749f5b/output/powerpc64le-linux-gnu/bitcoin-9b64d1749f5b-powerpc64le-linux-gnu-debug.tar.gz
2d94840ccfe6bb09ab0aaf51d9d54ed0cbd66eb8957554ae7dcfecb70a27fd1b  guix-build-9b64d1749f5b/output/powerpc64le-linux-gnu/bitcoin-9b64d1749f5b-powerpc64le-linux-gnu.tar.gz
cfb42ec938809247195d57be6a9f5219464f9a4dbaf3ce2cd74d732e3b87935f  guix-build-9b64d1749f5b/output/riscv64-linux-gnu/SHA256SUMS.part
a61fb13768962f1dd3b76186f604a9b29cd024a7f9fed50471f6e2bd5431bf0c  guix-build-9b64d1749f5b/output/riscv64-linux-gnu/bitcoin-9b64d1749f5b-riscv64-linux-gnu-debug.tar.gz
bac86b9c157ce510bca4b737356b80133f0948feac4ed7b98ca03d4faa83b90e  guix-build-9b64d1749f5b/output/riscv64-linux-gnu/bitcoin-9b64d1749f5b-riscv64-linux-gnu.tar.gz
120cefb379ba0bbd88711dcce978a3168ac5a0044267ca67cc7d8dcd4a97a59c  guix-build-9b64d1749f5b/output/x86_64-apple-darwin18/SHA256SUMS.part
e2ac86fca93b2b066752a6cf1f4af36c037e7f3ac3b094e71ce12b0892c97aec  guix-build-9b64d1749f5b/output/x86_64-apple-darwin18/bitcoin-9b64d1749f5b-osx-unsigned.dmg
fdc27376db1fc1a10fce9c4ab19da47bd2693fefa7cf0716ca89e22793d07d32  guix-build-9b64d1749f5b/output/x86_64-apple-darwin18/bitcoin-9b64d1749f5b-osx-unsigned.tar.gz
2dea8736bbd9881bd5f2bb4303390d6be06fef1d6cb4669760361190e8a91976  guix-build-9b64d1749f5b/output/x86_64-apple-darwin18/bitcoin-9b64d1749f5b-osx64.tar.gz
a1bd5a21ac1314d74d99da6fac6609f5dc282bdac54cce94dd2722cccc401c9e  guix-build-9b64d1749f5b/output/x86_64-linux-gnu/SHA256SUMS.part
998fe0dd4b13ab7b930c58788e81be5c42841d61d37bdf9ad26607e84b285fdd  guix-build-9b64d1749f5b/output/x86_64-linux-gnu/bitcoin-9b64d1749f5b-x86_64-linux-gnu-debug.tar.gz
c4b8ad245a3ea230f1514d416b4c9f9ad12a7d17a800f63697e1c93271ae3df5  guix-build-9b64d1749f5b/output/x86_64-linux-gnu/bitcoin-9b64d1749f5b-x86_64-linux-gnu.tar.gz
dc7a5ca9fd16017330f2f668618342e3088e464b10381bd10763f85307d53b3f  guix-build-9b64d1749f5b/output/x86_64-w64-mingw32/SHA256SUMS.part
b4245aba901c9a223867d17cc440e0e6a15c6b5ec2934d2c149076235ae0efb7  guix-build-9b64d1749f5b/output/x86_64-w64-mingw32/bitcoin-9b64d1749f5b-win-unsigned.tar.gz
8005a4dc58c92c6e2b6018f62cc960e2b5fdd600f88fbfea10bb7fd4123c12ae  guix-build-9b64d1749f5b/output/x86_64-w64-mingw32/bitcoin-9b64d1749f5b-win64-debug.zip
1f5fcaea89614cd6107386e43a4679fd05ab0b0ce22ddb47d34313e65d275307  guix-build-9b64d1749f5b/output/x86_64-w64-mingw32/bitcoin-9b64d1749f5b-win64-setup-unsigned.exe
dcd8814debfc8dc7a90c86572fd753afea0c772ac34976692118af626b41d117  guix-build-9b64d1749f5b/output/x86_64-w64-mingw32/bitcoin-9b64d1749f5b-win64.zip
```

fanquake commented on depends/config.site.in:
```diff
@@ -81,7 +81,7 @@ PKG_CONFIG="$(which pkg-config) --static"
 # These two need to remain exported because pkg-config does not see them
 # otherwise. That means they must be unexported at the end of configure.ac to
 # avoid ruining the cache. Sigh.
-export PKG_CONFIG_PATH="${depends_prefix}/share/pkgconfig:${depends_prefix}/lib/pkgconfig"
```
In be1f13107d561f4a9601515aff1d74bdf650bbd7:
> share/pkgconfig directory is a wrong place to search for *.pc files.

Why is it the wrong place look? The `pkg-config` man page says `On most systems, pkg-config looks in /usr/lib/pkgconfig, /usr/share/pkgconfig, /usr/local/lib/pkgconfig and /usr/local/share/pkgconfig for these files.`

hebasto replied:
> In [be1f131](https://github.com/bitcoin/bitcoin/commit/be1f13107d561f4a9601515aff1d74bdf650bbd7):
> 
> > share/pkgconfig directory is a wrong place to search for *.pc files.
> 
> Why is it the wrong place look? The `pkg-config` man page says `On most systems, pkg-config looks in /usr/lib/pkgconfig, /usr/share/pkgconfig, /usr/local/lib/pkgconfig and /usr/local/share/pkgconfig for these files.`

You're right. I erroneously relied on the online `pkg-config` [man page](https://linux.die.net/man/1/pkg-config) which missed the line you cited. And it is confirmed by the `pkg-config` source code I checked out.

Would "share/pkgconfig directory is a not place where dependency package *.pc files reside" a goog correction?

hebasto commented:
Updated 2d79a67b65b0f4b3cb1dd424707845f6bc89e64b -> 539ca409c939a31bc51f41f14ebb8bf8f48e0073 ([pr22783.02](https://github.com/hebasto/bitcoin/commits/pr22783.02) -> [pr22783.03](https://github.com/hebasto/bitcoin/commits/pr22783.03), [diff](https://github.com/hebasto/bitcoin/compare/pr22783.02..pr22783.03)):

- addressed @fanquake's [comment](https://github.com/bitcoin/bitcoin/pull/22783#discussion_r725989058)

#### Guix builds:
```
$ find guix-build-$(git rev-parse --short=12 HEAD)/output/ -type f -print0 | env LC_ALL=C sort -z | xargs -r0 sha256sum
9f84d58e4050ac2c5988b29667c6f7c529f36372413e590450be6ef39a6709f4  guix-build-539ca409c939/output/aarch64-linux-gnu/SHA256SUMS.part
7c12d928a7ed8a719800ad4b7091d252daa4972003c284137d38063f0a41725a  guix-build-539ca409c939/output/aarch64-linux-gnu/bitcoin-539ca409c939-aarch64-linux-gnu-debug.tar.gz
4b05cec6f230990be028519ccbb3b401e962573a1588d3a2a1cf6c4c38a9e586  guix-build-539ca409c939/output/aarch64-linux-gnu/bitcoin-539ca409c939-aarch64-linux-gnu.tar.gz
0a03840c14e6e02845fc4f0122e764b6f579f39139b10b0a29d9979da2a621bf  guix-build-539ca409c939/output/arm-linux-gnueabihf/SHA256SUMS.part
86d5e87a047be3cbe2c17cbfac4730ab59dd18ca829ab13029db7e499d8e2a85  guix-build-539ca409c939/output/arm-linux-gnueabihf/bitcoin-539ca409c939-arm-linux-gnueabihf-debug.tar.gz
b8611772d48ce6e0b59e20d041d50db964f976a1a5e631bcf8b2211044e97b4f  guix-build-539ca409c939/output/arm-linux-gnueabihf/bitcoin-539ca409c939-arm-linux-gnueabihf.tar.gz
11bb12b7b69819296a6a7d01e2ccec05ac51a4c274996b8efe5f859709241e0c  guix-build-539ca409c939/output/dist-archive/bitcoin-539ca409c939.tar.gz
bddd1bd7f76746e257e7c7b594117adc03e5f2bba0e2cd5c59d10d235894e0b4  guix-build-539ca409c939/output/powerpc64-linux-gnu/SHA256SUMS.part
c1ea0f371fba824606352b864ffd6c542e1a534902ce16000bb240e5401265ce  guix-build-539ca409c939/output/powerpc64-linux-gnu/bitcoin-539ca409c939-powerpc64-linux-gnu-debug.tar.gz
7f584df132fcbfd52a093ed3052a053b7772af875f1b223a7e01eeae39d78b64  guix-build-539ca409c939/output/powerpc64-linux-gnu/bitcoin-539ca409c939-powerpc64-linux-gnu.tar.gz
796d597be950cf3bb04d168b2714c5195db025f73aec9589dfb65c92e67bd9ec  guix-build-539ca409c939/output/powerpc64le-linux-gnu/SHA256SUMS.part
194406402cbe4785a33e3b0a8b87f143fb3c0fef040b69c0b58a354cd652cc54  guix-build-539ca409c939/output/powerpc64le-linux-gnu/bitcoin-539ca409c939-powerpc64le-linux-gnu-debug.tar.gz
e7d72347031d9aab09b0fc11a9ed6a6d9499aa02edb5605520131249aabaf8fa  guix-build-539ca409c939/output/powerpc64le-linux-gnu/bitcoin-539ca409c939-powerpc64le-linux-gnu.tar.gz
6f3b7368b48d50ad78f62c385bd2c3a1ca27640de5bbcef0306037e9989a6c67  guix-build-539ca409c939/output/riscv64-linux-gnu/SHA256SUMS.part
f89d8b4acc65ce0b228d427bf69d0ec4a714cdea7408bdba1613282cb717fbd1  guix-build-539ca409c939/output/riscv64-linux-gnu/bitcoin-539ca409c939-riscv64-linux-gnu-debug.tar.gz
fd182ed69eb805c7062d232ccfdedf04dd1b5473e4f56546fca064fc1fc0f8f9  guix-build-539ca409c939/output/riscv64-linux-gnu/bitcoin-539ca409c939-riscv64-linux-gnu.tar.gz
7c11a78e44ca493078271710287b4a4e9ca34b4052aa4758368987b73d3f8724  guix-build-539ca409c939/output/x86_64-apple-darwin18/SHA256SUMS.part
4f8a03e4a73bc4255e44fd5fdea9aabb2f9cce6e483b56f2fd4e84ba1312a044  guix-build-539ca409c939/output/x86_64-apple-darwin18/bitcoin-539ca409c939-osx-unsigned.dmg
1e583af565f3e49f755ba94de07616e530567dab5ec0f13f88ea4ff845614237  guix-build-539ca409c939/output/x86_64-apple-darwin18/bitcoin-539ca409c939-osx-unsigned.tar.gz
5a8ae0dea7c92f3af35ae20ccb2b1bc0cc654e0502822bedd269cecada4919b9  guix-build-539ca409c939/output/x86_64-apple-darwin18/bitcoin-539ca409c939-osx64.tar.gz
5ac3302f7e23b3647a94584e968f15d9ead69c78db711ae0541edc6c0c59ab95  guix-build-539ca409c939/output/x86_64-linux-gnu/SHA256SUMS.part
a38d33c7bf1dc853421e475052f0e3e160c7218695cac5074d0b26dbd397dade  guix-build-539ca409c939/output/x86_64-linux-gnu/bitcoin-539ca409c939-x86_64-linux-gnu-debug.tar.gz
a7a400029e59902a90469a4812a17b5003ad497dfdb7185cb75d42a12737c9c0  guix-build-539ca409c939/output/x86_64-linux-gnu/bitcoin-539ca409c939-x86_64-linux-gnu.tar.gz
717eac3cdfb90acd81c32a124a3eda7cf34b96607a52ed3f7f53c377c26f80af  guix-build-539ca409c939/output/x86_64-w64-mingw32/SHA256SUMS.part
a374de7280be7091ab02b271172929857a8971b7c8642284f1e0b54e78549b3e  guix-build-539ca409c939/output/x86_64-w64-mingw32/bitcoin-539ca409c939-win-unsigned.tar.gz
c2ee9470347d73daf56ae4274f593a8e2d81de8bd8702a6a1be6d537673be33d  guix-build-539ca409c939/output/x86_64-w64-mingw32/bitcoin-539ca409c939-win64-debug.zip
8a8e6191ff86323c024882fa91fbb967634623809b579f3f89c17aaaa9339f17  guix-build-539ca409c939/output/x86_64-w64-mingw32/bitcoin-539ca409c939-win64-setup-unsigned.exe
2ae82b855421f3041c470c4e166bfcb39d2a1b3912ef7e0dadd8f4e7227ded3d  guix-build-539ca409c939/output/x86_64-w64-mingw32/bitcoin-539ca409c939-win64.zip
```

fanquake reviewed (approved):
ACK 539ca409c939a31bc51f41f14ebb8bf8f48e0073

Guix Build:
```bash
bash-5.1# find guix-build-$(git rev-parse --short=12 HEAD)/output/ -type f -print0 | env LC_ALL=C sort -z | xargs -r0 sha256sum
9f84d58e4050ac2c5988b29667c6f7c529f36372413e590450be6ef39a6709f4  guix-build-539ca409c939/output/aarch64-linux-gnu/SHA256SUMS.part
7c12d928a7ed8a719800ad4b7091d252daa4972003c284137d38063f0a41725a  guix-build-539ca409c939/output/aarch64-linux-gnu/bitcoin-539ca409c939-aarch64-linux-gnu-debug.tar.gz
4b05cec6f230990be028519ccbb3b401e962573a1588d3a2a1cf6c4c38a9e586  guix-build-539ca409c939/output/aarch64-linux-gnu/bitcoin-539ca409c939-aarch64-linux-gnu.tar.gz
0a03840c14e6e02845fc4f0122e764b6f579f39139b10b0a29d9979da2a621bf  guix-build-539ca409c939/output/arm-linux-gnueabihf/SHA256SUMS.part
86d5e87a047be3cbe2c17cbfac4730ab59dd18ca829ab13029db7e499d8e2a85  guix-build-539ca409c939/output/arm-linux-gnueabihf/bitcoin-539ca409c939-arm-linux-gnueabihf-debug.tar.gz
b8611772d48ce6e0b59e20d041d50db964f976a1a5e631bcf8b2211044e97b4f  guix-build-539ca409c939/output/arm-linux-gnueabihf/bitcoin-539ca409c939-arm-linux-gnueabihf.tar.gz
11bb12b7b69819296a6a7d01e2ccec05ac51a4c274996b8efe5f859709241e0c  guix-build-539ca409c939/output/dist-archive/bitcoin-539ca409c939.tar.gz
bddd1bd7f76746e257e7c7b594117adc03e5f2bba0e2cd5c59d10d235894e0b4  guix-build-539ca409c939/output/powerpc64-linux-gnu/SHA256SUMS.part
c1ea0f371fba824606352b864ffd6c542e1a534902ce16000bb240e5401265ce  guix-build-539ca409c939/output/powerpc64-linux-gnu/bitcoin-539ca409c939-powerpc64-linux-gnu-debug.tar.gz
7f584df132fcbfd52a093ed3052a053b7772af875f1b223a7e01eeae39d78b64  guix-build-539ca409c939/output/powerpc64-linux-gnu/bitcoin-539ca409c939-powerpc64-linux-gnu.tar.gz
796d597be950cf3bb04d168b2714c5195db025f73aec9589dfb65c92e67bd9ec  guix-build-539ca409c939/output/powerpc64le-linux-gnu/SHA256SUMS.part
194406402cbe4785a33e3b0a8b87f143fb3c0fef040b69c0b58a354cd652cc54  guix-build-539ca409c939/output/powerpc64le-linux-gnu/bitcoin-539ca409c939-powerpc64le-linux-gnu-debug.tar.gz
e7d72347031d9aab09b0fc11a9ed6a6d9499aa02edb5605520131249aabaf8fa  guix-build-539ca409c939/output/powerpc64le-linux-gnu/bitcoin-539ca409c939-powerpc64le-linux-gnu.tar.gz
6f3b7368b48d50ad78f62c385bd2c3a1ca27640de5bbcef0306037e9989a6c67  guix-build-539ca409c939/output/riscv64-linux-gnu/SHA256SUMS.part
f89d8b4acc65ce0b228d427bf69d0ec4a714cdea7408bdba1613282cb717fbd1  guix-build-539ca409c939/output/riscv64-linux-gnu/bitcoin-539ca409c939-riscv64-linux-gnu-debug.tar.gz
fd182ed69eb805c7062d232ccfdedf04dd1b5473e4f56546fca064fc1fc0f8f9  guix-build-539ca409c939/output/riscv64-linux-gnu/bitcoin-539ca409c939-riscv64-linux-gnu.tar.gz
7c11a78e44ca493078271710287b4a4e9ca34b4052aa4758368987b73d3f8724  guix-build-539ca409c939/output/x86_64-apple-darwin18/SHA256SUMS.part
4f8a03e4a73bc4255e44fd5fdea9aabb2f9cce6e483b56f2fd4e84ba1312a044  guix-build-539ca409c939/output/x86_64-apple-darwin18/bitcoin-539ca409c939-osx-unsigned.dmg
1e583af565f3e49f755ba94de07616e530567dab5ec0f13f88ea4ff845614237  guix-build-539ca409c939/output/x86_64-apple-darwin18/bitcoin-539ca409c939-osx-unsigned.tar.gz
5a8ae0dea7c92f3af35ae20ccb2b1bc0cc654e0502822bedd269cecada4919b9  guix-build-539ca409c939/output/x86_64-apple-darwin18/bitcoin-539ca409c939-osx64.tar.gz
5ac3302f7e23b3647a94584e968f15d9ead69c78db711ae0541edc6c0c59ab95  guix-build-539ca409c939/output/x86_64-linux-gnu/SHA256SUMS.part
a38d33c7bf1dc853421e475052f0e3e160c7218695cac5074d0b26dbd397dade  guix-build-539ca409c939/output/x86_64-linux-gnu/bitcoin-539ca409c939-x86_64-linux-gnu-debug.tar.gz
a7a400029e59902a90469a4812a17b5003ad497dfdb7185cb75d42a12737c9c0  guix-build-539ca409c939/output/x86_64-linux-gnu/bitcoin-539ca409c939-x86_64-linux-gnu.tar.gz
717eac3cdfb90acd81c32a124a3eda7cf34b96607a52ed3f7f53c377c26f80af  guix-build-539ca409c939/output/x86_64-w64-mingw32/SHA256SUMS.part
a374de7280be7091ab02b271172929857a8971b7c8642284f1e0b54e78549b3e  guix-build-539ca409c939/output/x86_64-w64-mingw32/bitcoin-539ca409c939-win-unsigned.tar.gz
c2ee9470347d73daf56ae4274f593a8e2d81de8bd8702a6a1be6d537673be33d  guix-build-539ca409c939/output/x86_64-w64-mingw32/bitcoin-539ca409c939-win64-debug.zip
8a8e6191ff86323c024882fa91fbb967634623809b579f3f89c17aaaa9339f17  guix-build-539ca409c939/output/x86_64-w64-mingw32/bitcoin-539ca409c939-win64-setup-unsigned.exe
2ae82b855421f3041c470c4e166bfcb39d2a1b3912ef7e0dadd8f4e7227ded3d  guix-build-539ca409c939/output/x86_64-w64-mingw32/bitcoin-539ca409c939-win64.zip
```
